src/Event/RequestEvent.php

```diff
@@ -12,17 +12,11 @@
  */
 class RequestEvent
 {
-    private $microservice;
-    private $method;
-    private $uri;
-    private $options;
-
-    public function __construct(Microservice $microservice, string $method, string $uri, array $options)
+    public function __construct(private Microservice $microservice,
```

Comment:
```suggestion
    public function __construct(
        private Microservice $microservice,
```
Same for other occurrences